modules/product/src/Form/AddToCartForm.php

```diff
@@ -190,54 +215,47 @@ protected function variationElement(array $form, FormStateInterface $form_state)
         '#type' => 'value',
         '#value' => $selected_variation->id(),
       ];
-      return $form;
     }
-
-    // Build the full attribute form.
-    $selected_variation = $this->selectVariationFromUserInput($variations, $form_state);
-    $form['variation'] = [
-      '#type' => 'value',
-      '#value' => $selected_variation->id(),
-    ];
-    $form['attributes'] = [
-      '#type' => 'container',
-      '#attributes' => [
-        'class' => ['attribute-widgets'],
-      ],
-    ];
-    foreach ($this->getAttributeInfo($selected_variation, $variations) as $field_name => $attribute) {
-      $form['attributes'][$field_name] = [
-        '#type' => $attribute['type'],
-        '#title' => $attribute['title'],
-        '#options' => $attribute['values'],
-        '#required' => $attribute['required'],
-        '#default_value' => $selected_variation->get($field_name)->target_id,
-        '#ajax' => [
-          'callback' => '::ajaxRefresh',
-          'wrapper' => $form['#wrapper_id'],
+    else {
```

Comment:
This entire change seems to be style-only (moving the code under the else)? Wrong PR for that (and I'd disagree on the style choice, cause the cause was to lower the number of indentation levels)